src/js/components/Layer/LayerContainer.js

```diff
@@ -124,7 +124,10 @@ const LayerContainer = forwardRef(
         // determine which portal id the target is in, if any
         let clickedPortalId = null;
         let node =
-          containerTarget === document.body ? event.target : event?.path[0];
+          containerTarget === document.body
+            ? event.target
+            : event?.composedPath();
```

Comment:
Just one more thing...
```suggestion
            : event?.composedPath()[0];
```